agent/fw_laravel_queue.c

```diff
@@ -650,13 +682,13 @@ NR_PHP_WRAPPER(nr_laravel_queue_worker_process) {
   nr_php_arg_release(&job);
 
   /*
-   * End the real transaction and then start a new transaction so our
+   * End the useless transaction and then start a new transaction so our
    * instrumentation continues to fire, knowing that we'll ignore that
-   * transaction either when Worker::process() is called again or when
-   * WorkCommand::handle() exits.
+   * transaction if the worker continues when Worker::process() is called again.
    */
-  nr_php_txn_end(0, 0 TSRMLS_CC);
+  nr_php_txn_end(1, 0 TSRMLS_CC);
   nr_php_txn_begin(NULL, NULL TSRMLS_CC);
+  NR_PHP_WRAPPER_CALL;
```

Comment:
Why was this changed? `nr_laravel_queue_worker_process` is only used for PHPs < 8.0 and NR_PHP_WRAPPER_CALL must be called earlier for the job to actually execute and for the agent to capture job's execution as background transaction.